packages/sdk/src/tasks/finalize-withdrawal.ts

```diff
@@ -147,8 +170,12 @@ task('finalize-withdrawal', 'Finalize a withdrawal')
       clearInterval(finalizeInterval)
     }
 
-    const tx = await messenger.finalizeMessage(txHash)
-    receipt = await tx.wait()
-    console.log(receipt)
-    console.log('Finalized withdrawal')
+    try {
+      const tx = await messenger.finalizeMessage(txHash)
+      receipt = await tx.wait()
+      console.log(receipt)
+      console.log('Finalized withdrawal')
+    } catch (e) {
+      console.log(`Error occurred during finalizing withdrawal: ${e}`)
```

Comment:
Could you add an error message that `If you run this script on testnet, in most case, this happens when relayer has finalized the transaction already.`